scss/misc.scss

```
@@ -105,3 +107,35 @@ input[type="color"]:focus,
 .period-chooser-header {
   border-radius: 0;
 }
+
+// compatibility with the top contributors sidebar theme component
+// https://meta.discourse.org/t/top-contributors-sidebar/215110
+.list-container
+  #list-area
+  > .contents
+  > .topic-list:has(.discourse-top-contributors) {
+  grid-template-areas: "head head" "body sidebar";
+  grid-template-rows: auto 1fr;
+
+  @include viewport.until(lg) {
+    grid-template-areas: "head head" "body body";
+  }
+
+  tbody {
+    display: flex;
+  }
+
+  .discourse-top-contributors {
+    width: auto;
+    box-sizing: border-box;
+
+    .top-contributors-heading {
+      font-size: var(--font-up-2);
+      padding-bottom: 0.5em;
+    }
+
+    @include viewport.until(lg) {
+      display: none;
+    }
```

```suggestion
    width: auto;
    box-sizing: border-box;

    @include viewport.until(lg) {
      display: none;
    }

    .top-contributors-heading {
      font-size: var(--font-up-2);
      padding-bottom: 0.5em;
    }
```

btw, we don't have it codified anywhere (yet) but the preferred order would probably be something like:

props
conditional props (/mixins)
variants
pseudo-elements
nested elements

e.g.

```scss
color: black;

@include viewport.until(lg) {
  color: blue;
}

&:last-child, &.unread {
  color: red;
}

&::after {
  content: "foo";
}

.avatar img {
  padding: 1rem;
}
```